app/helpers/questions_helper.rb

```diff
@@ -18,15 +18,17 @@ def next_question(current_question)
     if current_question_index.nil?
       questions_to_ask.first
     elsif current_question_index == (questions_to_ask.length - 1)
-      RESULTS_PAGE
+      FINAL_QUESTION
     else
       questions_to_ask[current_question_index + 1]
     end
   end
 
   def previous_question(current_question)
     current_question_index = questions_to_ask.index(current_question)
-    if current_question_index.zero?
+    if current_question_index.nil?
+      questions_to_ask.last
```

Comment:
What is this doing?

Reply:
Allowing us to populate the Back link on the "are you able to leave home" question

Reply:
So it's basically checking if current_question == FINAL_QUESTION?

Reply:
No.  If the current question (i.e. `able_to_leave`) is not in `questions_to_ask`, then `.index` will return `nil`, in which case we should use the last question on the back button.

Reply:
But you could also do it as `return questions_to_ask.last if current_question == FINAL_QUESTION?` before doing all the stuff with `current_question_index`?

Reply:
That's true.  I'll update this.